### ISSUE TITLE
"aria-labelledby" references are referring to IDs that do not exist

### ISSUE THREAD

**Issue by venkatx5** [microsoft/BotFramework-WebChat#3676]
We are using webchat-es5.js for our Virtual assistant and our customer evaluating for 508 compliance. The following WAI-ARIA 'aria-labelledby' attribute has a reference to an ID that does not exist (broken reference) on the Bot response:

•webchat__stacked-layout__id--fxqmh
•webchat__stacked-layout__id--ok1u7

The references generated for user entered text has valid references. Issue is only on the bot generated responses. As per the ARIA guideline, the referenced id must present. Need assistance on this to be 508 compliant.

## Screenshots
In User Text, (This is fine)
![image](https://user-images.githubusercontent.com/11777507/105128574-4aa39580-5b09-11eb-930d-9a809e5e9443.png)
In Bot response, the reference is missing.
![image](https://user-images.githubusercontent.com/11777507/105127935-ed5b1480-5b07-11eb-8acd-6c936c04e02a.png)

## Version
webchat es5 version 4.11.0
https://cdn.botframework.com/botframework-webchat/4.11.0/webchat-es5.js
Integrating the CDN version js directly on the page using script tag (No Iframe). Desktop Chrome Browser.

```
<meta name="botframework-directlinespeech:version" content="4.11.0">
<meta name="botframework-webchat:bundle:variant" content="full-es5">
<meta name="botframework-webchat:bundle:version" content="4.11.0">
<meta name="botframework-webchat:core:version" content="4.11.0">
<meta name="botframework-webchat:ui:version" content="4.11.0">
```

## Describe the bug
We are using webchat-es5.js for our Virtual assistant and our customer evaluating for 508 compliance. The WAI-ARIA 'aria-labelledby' attribute has a reference to an ID that does not exist (broken reference) on the Bot response. In user input text, the reference is proper. To be compliant, the bot response broken reference should be resolved.

## Steps to reproduce
1. Deploy Web App Bot with adaptive cards.
2. Open the web app bot on browser.
4. Interact with the bot till the adaptive card response arrives.
5. Search for "aria-labelledby" to get the reference value "Eg. webchat__stacked-layout__id--xv8x4 ".
6. Search for the reference "webchat__stacked-layout__id--fxqmh". A div should be present with id="webchat__stacked-layout__id--fxqmh".

## Expected behavior
For every  "aria-labelledby" reference, A div should be present with id="webchat__stacked-layout__id--fxqmh".

[Bug]

**Comment by corinagum** [comment]
Thanks for reporting. I'm adding this to our list for triage.

**Comment by venkatx5** [comment]
Any updates/fixes that we can try?

**Comment by compulim** [comment]
We have a fix in PR review now. It is expected to ship around February 15.

**Comment by venkatx5** [comment]
> We have a fix in PR review now. It is expected to ship around February 15.

That's wonderful. Thanks a lot for the Swift action. 
Hope it'll be webchat version 12.0?

**Comment by corinagum** [comment]
> > We have a fix in PR review now. It is expected to ship around February 15.
> 
> That's wonderful. Thanks a lot for the Swift action.
> Hope it'll be webchat version 12.0?

Yes, 4.12 is slated for March 8th release.

**Comment by venkatx5** [comment]
> > > We have a fix in PR review now. It is expected to ship around February 15.
> > 
> > 
> > That's wonderful. Thanks a lot for the Swift action.
> > Hope it'll be webchat version 12.0?
> 
> Yes, 4.12 is slated for March 8th release.

 May I know what's available on February 15?
